commcare_export/env.py

```diff
@@ -222,9 +222,17 @@ def emitted_tables(self):
 
 @unwrap('val')
 def str2bool(val):
+    if val is None:
+        return False
+
     if isinstance(val, bool):
         return val
-    return val and str(val).lower() in {'true', 't', '1'}
+
+    if not isinstance(val, six.string_types):
+        val = str(val)
```

Comment:
Since the data is coming from HQ API's which should always be `utf-8` encoded I'm going to go with `val.decode('utf-8')`.